### ISSUE TITLE
Exit code 3221225477 - IsSandboxedProcess / GetHandleVerifier

### ISSUE THREAD

**Issue by fasterthanlime** [electron/electron#9646]
* Electron version: 1.7.2
* Operating system: Windows 10.0.14393 Build 14393

### Expected behavior

Electron should hum happily.

### Actual behavior

Electron crashes with code 3221225477.

### How to reproduce

Well:

```
$ git clone https://github.com/itchio/itch -b get-handle-verifier
$ yarn
$ NODE_ENV=production yarn start
```

...but, it's a branch in the middle of a big rewrite, so it's not very reproduce-friendly. Plus, you need an itch.io account. Happens to me when scrolling through my 300+ games library, I'll see if I can gather a better repro case.

```
Backtrace:
        GetHandleVerifier [0x00007FF7BC0F518B+1439451]
        IsSandboxedProcess [0x00007FF7BC366384+564612]
        GetHandleVerifier [0x00007FF7BC0EFFF4+1418564]
        GetHandleVerifier [0x00007FF7BC0DF7F6+1350982]
        GetHandleVerifier [0x00007FF7BC0F07F1+1420609]
        IsSandboxedProcess [0x00007FF7BC30DF96+203158]
        IsSandboxedProcess [0x00007FF7BC43B880+1438336]
        GetHandleVerifier [0x00007FF7BC03977C+670924]
        GetHandleVerifier [0x00007FF7BBFDB986+286422]
        GetHandleVerifier [0x00007FF7BBFDA470+281024]
        GetHandleVerifier [0x00007FF7BC03A951+675489]
        GetHandleVerifier [0x00007FF7BC03B114+677476]
        GetHandleVerifier [0x00007FF7BBFD799E+270062]
        IsSandboxedProcess [0x00007FF7BC3D9FA2+1038754]
        IsSandboxedProcess [0x00007FF7BC3DB4EA+1044202]
        IsSandboxedProcess [0x00007FF7BC68E53A+3874618]
        IsSandboxedProcess [0x00007FF7BC671264+3755108]
        GetHandleVerifier [0x00007FF7BC2DB57D+3431117]
        GetHandleVerifier [0x00007FF7BC2DB474+3430852]
        GetHandleVerifier [0x00007FF7BC08E6A0+1018864]
        (No symbol) [0x00007FF7BBCF8464]
        IsSandboxedProcess [0x00007FF7BEF8B1AF+46853039]
        BaseThreadInitThunk [0x00007FFE61188364+20]
        RtlUserThreadStart [0x00007FFE63A970D1+33]
error Command failed with exit code 3221225477.
```

### Notes

I've been keeping an eye on #3570 - it's the same error code (but it's a pretty generic one). I *do* have some require extension at play here since I'm using https://github.com/electron/electron-compile - so there's something in common between both issues at least.

**Comment by fasterthanlime** [comment]
Another tidbit: I've recently started using [dtf](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat) to format dates - I see ICU in #3570, so, it might be related?

**Comment by fasterthanlime** [comment]
Switched to Linux for day-to-day development since yesterday and haven't seen that bug pop up (exact same electron version, exact same codebase), so I think it's windows-specific.

**Comment by fasterthanlime** [comment]
Saw that issue in the wild again, against electron@1.6.15

But I haven't seen it in a long while in a branch running against electron@1.7.8

A VSCode sister issue ( https://github.com/Microsoft/vscode/issues/31802 ) also seems to have been resolved between VSCode 1.15.0 (electron@1.6.6) and VSCode 1.17.2 (electron@1.7.7).

So, I'm assuming this issue has been fixed at some point in the 1.7.x branch. Closing! 🌟
